src/controllers/commands/CommandController.cpp

```diff
@@ -2829,6 +2829,54 @@ void CommandController::initialize(Settings &, Paths &paths)
         return "";
     });
 
+    this->registerCommand("/banid", [formatBanTimeoutError](
+                                        const QStringList &words,
+                                        auto channel) {
+        auto *twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+        if (twitchChannel == nullptr)
+        {
+            channel->addMessage(makeSystemMessage(
+                QString("The /banid command only works in Twitch channels")));
+            return "";
+        }
+
+        const auto *usageStr =
+            "Usage: \"/banid <userID> [reason]\" - Permanently prevent a user "
+            "from chatting via their userID. Reason is optional and will be "
+            "shown to the target user and other moderators. Use \"/unban "
```

Comment:
Does `/unban` work for usernames that are already banned? If not, we should not mention `/unban` here imo, not unless we have an `/unbanid` equivalent